pipelines/metrics_pipeline.yaml

```diff
@@ -2,7 +2,7 @@ description: Complete metrics measurement pipeline
 imports:
   - location: $FARO_DIR/pipelines/metrics_pipeline_matched.yaml
   - location: $FARO_DIR/pipelines/metrics_pipeline_matched_multi.yaml
-  - location: $FARO_DIR/pipelines/metrics_pipeline_visit.yaml
+  # - location: $FARO_DIR/pipelines/metrics_pipeline_visit.yaml
```

Comment:
We shouldn't leave commented out code, even in yaml files, I don't think.

Reply:
👍🏻

Reply:
Agreed - I think (and Dan T.'s questions in the thread on #dm-hsc-reprocessing are reminding me) that we need to settle on a standard set of pipelines, and add documentation (perhaps in the .yaml itself?) to explain what setting each pipeline is meant for. Otherwise we have a "main" pipeline in which we comment/uncomment things as needed.

Reply:
Will create a separate ticket to reorganize and improve documentation/readability of pipelines.